src/main/java/org/jitsi/jigasi/JvbConference.java

```diff
@@ -1649,6 +1654,27 @@ else if ("org.jitsi.jigasi.xmpp.acc.BOSH_URL_PATTERN".equals(overridenProp))
                     ctx.setBoshURL(value);
                 }
             }
+            else if ("org.jitsi.jigasi.xmpp.acc.USER_ID".equals(overridenProp)
+                && JigasiBundleActivator.getConfigurationService().getBoolean(overridePrefix + ".UNIQUE_USER_ID", false))
+            {
+                try
+                {
+                    Jid jid = JidCreate.from(value);
+                    long random = RANDOM.nextInt();
+                    Jid newJid = JidCreate.entityBareFrom(
+                            Localpart.from(
+                                    jid.getLocalpartOrNull() + "-" + String.format("%8h", random).replace(' ', '0')),
```

Comment:
Why do you need `.replace(' ', '0')`? Why "%h" (it calls hashCode) instead of %x?

Reply:
I used your code from here: https://github.com/jitsi/jigasi/blob/fb76d173eb643b94fb7ee10721af1a1647127ee9/src/main/java/org/jitsi/jigasi/CallContext.java#L460